packages/hardhat-cannon/src/tasks/verify.ts

```diff
@@ -50,18 +51,20 @@ task(TASK_VERIFY, 'Verify a package on Etherscan')
     }
 
     for (const c in outputs.contracts) {
-      console.log('Verifying contract:', c);
-      try {
-        await hre.run('verify:verify', {
-          contract: `${outputs.contracts[c].sourceName}:${outputs.contracts[c].contractName}`,
-          address: outputs.contracts[c].address,
-          constructorArguments: outputs.contracts[c].constructorArgs || [],
-        });
-      } catch (err) {
-        if ((err as Error).message.includes('Already Verified')) {
-          console.log('Already verified');
-        } else {
-          throw err;
+      if (_.get(outputs, ['contracts', c, 'sourceName']) && _.get(outputs, _.get(outputs, ['contracts', c, 'contractName'])) {
```

Comment:
there should be an `else` block which explains to the user that verification has been skipped for this contract because `sourceName / contractName for ${c} has not been set` or so

Reply:
also rather than wrapping everything in yet another deeper `if` block, why not just add a small `if` block at the beginning that checks if these properties exist if not, then `continue` out